### PR TITLE
Ledger txns

### PR DESCRIPTION
**Pull request by blackpanther1881**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,12 +2,18 @@
   "name": "pwallet",
   "private": true,
   "dependencies": {
+    "@cosmjs/amino": "^v0.25.4",
     "@cosmjs/crypto": "^v0.25.4",
+    "@cosmjs/encoding": "^v0.25.4",
+    "@cosmjs/launchpad": "^v0.25.4",
+    "@cosmjs/ledger-amino": "git://github.com/persistenceOne/ledger-amino.git#raju",
```
Comment (puneet2019) [inline package.json:9]: created a tag for this, can we use that instead?

File: src/ibcConfig.json [hunk 2]
```diff
@@ -3,14 +3,18 @@
...
+      "description": "cosmos test chain relayer, maintained by AUDIT.one"
     }
   ],
   "mainNetChannels": [
     {
       "name": "osmosis",
       "id": "osmosis/channel-6",
-      "port": "transfer"
+      "port": "transfer",
+      "url": "",
```
Comment (puneet2019) [inline src/ibcConfig.json:16]: can add osmosis url rpc.osmosis-1.audit.one

File: src/utils/transactions.js [hunk 3]
```diff
@@ -149,24 +174,26 @@ function decodeTendermintConsensusStateAny(consensusState) {
     }
     return tendermint_1.ConsensusState.decode(consensusState.value);
 }
-async function MakeIBCTransferMsg(channel, fromAddress, toAddress, amount, timeoutHeight, timeoutTimestamp = config.timeoutTimestamp, denom = "uxprt", port = "transfer") {
+
+async function MakeIBCTransferMsg(channel, fromAddress, toAddress, amount, timeoutHeight, timeoutTimestamp = config.timeoutTimestamp, denom = "uxprt", url, port = "transfer") {
+    console.log(url, "example url");
```
Comment (puneet2019) [inline src/utils/transactions.js:179]: log

**Review by puneet2019 — APPROVED** [review]

File: src/utils/ledger.js [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+    console.log(firstAccount, "address", "persistence15pc0lktktdj37dw0rqjsptpqnhqvxhfydva96r");
+    const client = await SigningStargateClient.connectWithSigner(RPCURL, signer);
+    const fee = {
+        amount: coins(100, "uxprt"),
+        gas: String(2000000)
+    };
+
+    const result = await client.signAndBroadcast(
+        firstAccount.address,
+        [SendMsg(firstAccount.address,"persistence15pc0lktktdj37dw0rqjsptpqnhqvxhfydva96r",1221,"uxprt" )],fee,"ledger Tx",
```
Comment (deepanshutr) [inline src/utils/ledger.js:57]: hardcoded details

**Review by deepanshutr — APPROVED** [review]

**Pull request merged**